src/figuresClasses.ts

```diff
@@ -1,19 +1,82 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea: () => number;
 }
 
-export class Triangle {
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
 
+function belowZero(property:number): void {
+  if (property <= 0) {
+    throw new Error('Error: radius must be greater than 0!!!');
```

Comment:
you've used this function to the Triangle and if one of the sides of the triangle will be 0 or above user will see a message about radius? something wrong here